pkg/cloud/cloud.go

```diff
@@ -279,31 +628,150 @@ func (c *cloud) getVolumeId(ctx context.Context, volumeId string) (*string, erro
 	}
 }
 
-func isSnapshotNotFound(err error) bool {
-	if awsErr, ok := err.(awserr.Error); ok {
-		if awsErr.Code() == fsx.ErrCodeSnapshotNotFound {
-			return true
+func (c *cloud) parseDiskIopsConfiguration(input string) (*fsx.DiskIopsConfiguration, error) {
```

Comment:
Not sure if it makes sense for helper functions, even cloud-specific ones, to be in cloud rather than util. Dont have a strong opinion on this though

Reply:
Personally I prefer to have utils only hold functions that could be utilized universally. Since the helper functions returns FSx objects we would have to import that package in utils, thus increasing compilation time when they would only be used by the cloud package.

Reply:
> increasing compilation time when they would only be used by the cloud package.
good point, makes sense to leave them where they are then